docs/ReleaseProcess.md

```diff
@@ -25,41 +27,69 @@ eBPF for Windows, and how to service it later.
+
+## Publishing the Release to GitHub
+
+6. Go to the repo on GitHub and click on "`<Code>`" and click on right the "`Create a new release`" link.
+7. Click on the "`Choose a tag`" combo box and select the tag with version number for the release, as created earlier.
+8. Fill in the release title as "`vX.Y.Z`". Note "`Z`" must be `0` for the monthly release. Otherwise, it should be the patch number.
+9. Manually enter release notes or click "`Generate release notes`" and then edit as desired.
+10. Attach the `.msi`, the (non-redist) `.nupkg`, the `Build-x64-[Release|Debug].zip` and the `Build-x64-native-only-[Release|Debug].X.Y.Z.zip` build (from the CI/CD artifacts), by dropping them in the "`Attach binaries by dropping them here or selecting them.`" area. For example, the file list for `v0.11.0` should be:
+
+    - *Build-x64-native-only-Debug.0.11.0.zip*
+    - *Build-x64-native-only-Release.0.11.0.zip*
+    - *ebpf-for-windows.0.11.0.msi*
+    - *eBPF-for-Windows.0.11.0.nupkg*
+    - *Build-x64-Debug.zip*
+    - *Build-x64-Release.zip*
+
+11. Check the "`Set as a pre-release`" checkbox, unless the release is production-signed.
+12. Once the uploads are complete, click "`Publish release`".
+
+## Publishing the Release to NuGet.org
 
-    - `eBPF-for-Windows.X.Y.Z.nupkg`
-1. Submit the PR for review (from its draft state), and wait for it to be approved and merged into the main repo's "`main`" branch.
-1. On the main `ebpf-for-windows` repo, create a new release branch from `main` **corresponding to the previous PR's commit**, name it "`release/X.Y`" (replace "X.Y" with the version number being released).
-1. Publish the release as per the "[Publishing a Release](ReleaseProcess.md#publishing-a-release)" process.
+Upload the (**non-redist**) `.nupkg` file to [NuGet.org](https://www.nuget.org/) (the metadata inside the `.nuget` package itself will automatically populate all the other form fields).
 
-## Servicing a release
+## Servicing a Release
 
 Servicing a release has two main scenarios:
 
 ### Updating a Release branch with patches/hot-fixes from main (*Forward Integration*)
 
 >NOTE: In servicing a release branch, **new features must not be added to the release branch**.  Only patches or hot-fixes will be accepted.
 
-1. On the main `ebpf-for-windows` repo, create and check out a new working branch from the `/release/X.Y` branch you want to service.
-1. Run the following script from the root directory of the repository, within a "*Developer Poweshell for VS 2022"* instance. Make sure to follow [Semantic Versioning 2.0](https://semver.org) ("`X.Y.Z`"):
+1. On the upstream `ebpf-for-windows` repo, create and check out a new topic branch from the `release/X.Y` branch you want to service.
```

Comment:
Upstream (i.e. OneBranch?) should only be used for security sensitive fixes (i.e. MSRCs). Everything else should still go through GitHub.

Reply:
I meant the base/original repo. Will fix.